jax/numpy/__init__.pyi

```diff
@@ -410,6 +410,8 @@ def flipud(m: ArrayLike) -> Array: ...
 float16: Any
 float32: Any
 float64: Any
+float8_e3m4: Any
+float8_e4m3: Any
```

Comment:
I would remove these for now, since they may or may not actually be present in the namespace (and this can only be known at runtime)

Reply:
removed.